Makefile

```diff
@@ -2,35 +2,14 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-.PHONY: all test bootstrap install lint clean breakpad json_enhancements_pg_extension package
+.PHONY: package
 
-all: test
-
-test: bootstrap
-	./scripts/test.sh
-
-dev:
-	SOCORRO_DEVELOPMENT_ENV=1 ./scripts/bootstrap.sh
-
-bootstrap:
+# Used by infra to build rpm
+package:
```

Comment:
I collapsed the rules this required into this rule in the proper order. It's used by the socorro-infra for deploys and that's it.